tests/testServerWithStorage.js

```diff
@@ -317,11 +316,11 @@ sjrk.storyTelling.server.testServerWithStorageDefs = [{
     }, {
         event: "{storySave}.events.onComplete",
         listener: "sjrk.storyTelling.server.testServerWithStorageDefs.testStoryPostRequestSuccessful",
-        args: ["{arguments}.0", "{arguments}.1", "{that}.events.onStorySaveSuccessful"]
+        args: ["{arguments}.0", "{arguments}.1", "{that}.events.onStorySaveSuccessful", "{that}.configuration.server.options.globalConfig.savingEnabled"]
```

Comment:
The server test for the "savingEnabled" flag can be refactored better.

Right now, the "savingEnabled" value is dependent on what's set in the actual server source code "serverSetup.js", in which "savingEnabled" only has one value (true or false) at any moment. This means, running this test file only tests one flag value at any time rather than testing both cases. To fix it: 

1. No need to share the same set of test definition for 2 scenarios with a number of empty assertions. The current test definition is good for testing "true" scenario. Create a separate test definition for "false" scenario. In the new test, only responses from post story requests are checked to ensure an error is returned. The following checks for the saved story and the uploaded image are unnecessary.

2. An example of passing "savingEnabled" flag to the server component in test definitions:
```
sjrk.storyTelling.server.testServerWithStorageDefsWithSavingEnabled = [{
    name: "Test server with storage - savingEnabled is turned on",
    expect: 16,
    distributeOptions: {
        record: true,
        target: "{that server}.options.globalConfig.savingEnabled"
    }
...
}];
```
Please use a common variable to hold parts that can be shared by both definitions.